src/contracts/GPv2Settlement.sol

```diff
@@ -202,6 +202,17 @@ contract GPv2Settlement {
         outTransfer.owner = trade.owner;
         outTransfer.token = order.buyToken;
 
+        // NOTE: The following computation is derived from the equation:
+        // ```
+        // amount_x * price_x = amount_y * price_y
+        // ```
+        // Intuitively, if a chocolate bar is 0,50€ and a beer is 4€, 1 beer
```

Comment:
Very good comment, maybe we should use more intuitive descriptions like that in our comments